shared/enum.ts

```diff
@@ -0,0 +1,9 @@
+export enum ITokenEnum {
+  token = "token",
+  refreshToken = "refresh_token",
+  anonymousToken = "anonymousToken",
+  humanId = "human_id",
+  userId = "user_id",
+  targetUser = "target_user",
+  tempMessage = "temp_message",
```

Comment:
I think these do not represent TokenEnum.
Please move to another enum
e.g.
`LocalStorageKeyEnum`